### PR TITLE
48110 Direct users to FSS for Tracings

### PR DESCRIPTION
**Pull request by Shirin14926**
Code changes

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Index.cshtml
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Index.cshtml:25]: Please remove <td> tag as its not required here. Element 'td' cannot be nested inside element 'p'.
Also, please correct the alignment for the whole @if block.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml:24]: Please remove <td> tag as its not required here. Element 'td' cannot be nested inside element 'p'.
Also, please correct the alignment for the whole @if block.

**Review by usha13529 — CHANGES_REQUESTED** [review]
Minor comments to fix please.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml]: Removed and alignment made proper

**Review by usha13529 — APPROVED** [review]
Thanks for the changes @Shirin14926. Looks good to me.

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @Shirin14926 

Will wait until after Show & Tell for merge.

**Pull request merged**
